InteractionBase/src/handlers/GenericHandler.cpp

```diff
@@ -115,6 +120,9 @@ GenericHandlerManagerListener& GenericHandler::managerListener()
 
 GenericHandler::GenericHandler()
 {
+	supportedCommands.append(new NewViewCommand());
```

Comment:
Actually the correct place to add these commands would be to the SceneItemHandler